source/gui/__init__.py

```diff
@@ -367,7 +369,7 @@ def onInputGesturesCommand(self, evt):
 
 	def onAboutCommand(self, evt):
 		# Translators: The title of the dialog to show about info for NVDA.
-		messageBox(versionInfo.aboutMessage, _("About NVDA"), wx.OK)
```

Comment:
I would encourage changing messageBox to use MessageDialog to ensure the API is compatible. Same with the other message dialogs